examples/AerosolActivation/aerosol_activation.jl

```diff
@@ -0,0 +1,275 @@
+# # Aerosol Activation Example
+
+# ## Overview
+
+# This example is based on AerosolActivation module which is a part of the
+#   [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl) package.
+# The AerosolActivation module computes the total number and mass
+#   of aerosol particles that get activated and become cloud droplets,
+#   given the atmospheric conditions and
+#   the initial aerosol size distribution and properties.
+# See the AerosolActivation module
+#   [docs](https://clima.github.io/CloudMicrophysics.jl/dev/Aerosol_activation/)
+#   for derivation and description of all input parameters.
+#
+# In this example we use the ensemble Kalman methods to learn
+#   two parameters that describe the chemical composition of aerosol particles
+#   based on the observed total number and mass of activated particles.
+# The AerosolActivation model is used here in a "perfect model" setting,
+#   meaning that the observations are generated by the same module
+#   we are calibrating.
+
+# ## Prerequisites
+
+# The example depends on some standard Julia libraries,
+#   as well as the CliMA packages:
+#   [EnsembleKalmanProcess.jl](https://github.com/CliMA/EnsembleKalmanProcesses.jl),
+#   [CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl) and
+#   [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl).
+# To ensure that all the dependencies are met
+#   start Julia using the Project.toml file provided in the example
+#   and run the Julia package manager to download all the dependecies.
+
+# ## Example
+
+# We begin by importing some standard Julia modules,
+# the Ensemble Kalman Process modules,
+# CLIMA Parameter modules and Aerosol Activation modules.
+using Plots
+using Distributions
+using LinearAlgebra
+using Test
```

Comment:
```suggestion

```

No need to load `Test` just to test that two values are similar. You can just print them or print also their relative difference, i.e., `|value1 - value2| / |value1|`.